.github/workflows/issue-labeler.yml

```diff
@@ -0,0 +1,15 @@
+name: "Issue Labeler"
+on:
+  issues:
+    types: [opened, edited]
+
+jobs:
+  triage:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: github/issue-labeler@6ca237433dbbb8e475241b7f38f4600d9e296c57 # v2.5
+      with:
+        repo-token: "${{ secrets.GITHUB_TOKEN }}"
+        configuration-path: .github/issue-labeler-config.yml
```

Comment:
Can we move `.github/issue-labeler-config.yml` to `.github/workflows/issue-labeler-config.yml`?

Reply:
Moved it to `.github/workflows/issue-labeler/config.yml`